applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -378,11 +436,15 @@ config AURACAST
 
 menu "QoS"
 
-config BT_AUDIO_PRESENTATION_DELAY_US
-	int "Presentation delay"
+config BT_AUDIO_LOCAL_PRESENTATION_DELAY_US
+	int "Local, the audio source, presentation delay"
```

```suggestion
	int "Presentation delay"
```
The rest is explained in the help section